src/index.ts

```diff
@@ -1,16 +1,30 @@
-const paypal = require('paypal-rest-sdk');
+import * as paypal from "paypal-rest-sdk";
+
+export type reqData = {
+    method: string | undefined,
```

Comment:
Actually, he pointed me to something better: [if you add `?` to the parameter name it can be undefined](https://www.typescriptlang.org/docs/handbook/functions.html#optional-and-default-parameters), and that's built into TypeScript.